cranelift/codegen/src/isa/x86/abi.rs

```diff
@@ -70,6 +70,7 @@ struct Args {
     shared_flags: shared_settings::Flags,
     #[allow(dead_code)]
     isa_flags: isa_settings::Flags,
+    is_rets: bool,
```

Comment:
Nitpick: just to make this a little more clear for readers

```suggestion
    assigning_returns: bool,
```